.env.example

```diff
@@ -1,3 +1,5 @@
-# key must have length = 6 and contains only a-z characters
+# key must have length = 6 and contain only a-z characters
 CAPTCHA_ENCRYPTION_KEY=abcdef
```

Comment:
🤔

Reply:
When calling the captcha image, the path to it needs to include the challenge text (image="<url>/captcha?text=EXmlpL". However, the challenge would be pretty vulnerable if we just passed the right answer, because some bot would be able to read it. The solution I found for this is encrypting the challenge before sending it, so the right answer is not available in the routes